recipes/roaring/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "0.3.2":
```

Comment:
Note that supporting both 0.3 and 0.2 with the same build setup requires care. The 0.3 version is quite different, with runtime dispatching and so forth. Prior to 0.3, the person building the code under x64 needed some care to get max. performance. This is no longer needed under 0.3.

Reply:
Can you elaborate a bit on the "care" required?
The current method of building the roaring library is using the vanilla cmake build script.
We don't do anything special.

Reply:
I was making a reference to commit https://github.com/conan-io/conan-center-index/pull/6494/commits/fd0f203a6c14e217e7f944c732a5a92bab3eee9d where there was an attempt at setting architecture-specific flags. This could prove useful with version 0.2, but should now be unnecessary. One should just build without any special flag.